plugin/storage/cassandra/spanstore/writer.go

```diff
@@ -65,6 +65,13 @@ const (
 	durationBucketSize = time.Hour
 )
 
+const (
```

Comment:
above section is already "const", don't need to start a new one

Reply:
I wanted to reset the index of `iota`, and it seems that it only happens in a new `const` block according to https://golang.org/ref/spec#Iota